<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaixaOrcamento</class>
 <widget class="QDialog" name="BaixaOrcamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dar baixa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Motivo</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido_5">
          <property name="text">
           <string>Desistiu/Parou obra</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido_4">
          <property name="text">
           <string>Optou outro produto/concorrência</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido_2">
          <property name="text">
           <string>Preço/Concorrência</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido_3">
          <property name="text">
           <string>Prazo de entrega</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonCancelado">
          <property name="text">
           <string>Cancelado</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido">
          <property name="text">
           <string>Perdido</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QRadioButton" name="radioButtonPerdido_6">
          <property name="text">
           <string>Arquiteto</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Observação</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEditObservacao"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonCancelado</tabstop>
  <tabstop>radioButtonPerdido</tabstop>
  <tabstop>radioButtonPerdido_2</tabstop>
  <tabstop>radioButtonPerdido_3</tabstop>
  <tabstop>radioButtonPerdido_4</tabstop>
  <tabstop>radioButtonPerdido_5</tabstop>
  <tabstop>radioButtonPerdido_6</tabstop>
  <tabstop>plainTextEditObservacao</tabstop>
  <tabstop>pushButtonSalvar</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
